plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHiveInMemoryMetastore.java

```diff
@@ -67,4 +69,61 @@ public void testDisallowQueryingOfIcebergTables()
     {
         throw new SkipException("not supported");
     }
+
+    @Test
```

Comment:
Why not change InMemoryThriftMetastore like the other metastores instead of overriding tests ?

Reply:
All updates to `InMemoryThriftMetastore` are correct and sure. Only for consistency we could alter this to behave as others.